
As of version 3.8, GNU grep added warning messages whenever the
`egrep` or `fgrep` commands are used. While egrep and fgrep are not present
in the latest POSIX standard, there is no reason to do this to users and
it's inconsistent with our native egrep and fgrep utilities.

Index: grep/src/egrep.sh
===================================================================
diff -wpruN '--exclude=*.orig' a~/src/egrep.sh a/src/egrep.sh
--- a~/src/egrep.sh	1970-01-01 00:00:00
+++ a/src/egrep.sh	1970-01-01 00:00:00
@@ -1,4 +1,3 @@
 #!@SHELL@
 cmd=${0##*/}
-echo "$cmd: warning: $cmd is obsolescent; using @grep@ @option@" >&2
 exec @grep@ @option@ "$@"
